### PR TITLE
Balance pull requests no longer charge you 8 GBP.

### PR DESCRIPTION
**Pull request by Iamgoofball**
I get that we want to discourage I ded pls nerf PRs and all but sometimes we need to fix the game's balance and/or broken features and currently your options are to
1. Fix it, eat the -8 GBP
2. Remove it, GBP remains unchanged
3. beg a maintainer for GBP: No Update

all 3 of these options suck and we shouldn't be discouraging people trying to make the game not suck to play when we made big mistakes and merge in badly balanced crap
as an example, here's some things that needed fixed with the game balance that I was billed a grand total of **40 GBP** for:
1. making quadsec drinkable IRL(jesus christ creme de menthe was a mistake for that recipe) https://github.com/tgstation/tgstation/pull/64005
2. making tourettes no longer a 20 second hardstun after literally 15 goddamn years according to the svn logs https://github.com/tgstation/tgstation/pull/64416
3. making security not carry around 20 force sharp knives 24/7 https://github.com/tgstation/tgstation/pull/62870
5. making felinid tail grabbing not the literal best possible mood buff available https://github.com/tgstation/tgstation/pull/62768
6. making felinid water debuff not the literal worst possible mood debuff available https://github.com/tgstation/tgstation/pull/62769

these were all critical issues compromising the balance of my tastebuds/my sanity when suffering from tourettes ingame/security/mood and i was billed 40 GBP for them as if I had made 5 PRs arguing over whether security disablers should do 1 point of extra damage, because we never undid our negative GBP for balance changes from when people were making i ded balance PRs 24/7

# official maintainer request to open this PR:
Discord_GyqADDdarx

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
A problem I have with this being so low is making fix + balance basically negate itself. Balance should still cost GBP so I think a -4 or -5 max, would be better than current.

**Comment by Kylerace** [comment]
i hate balance discussions but balance prs are themselves not a threat to maintainability and (usually) short, i dont see why it should be -8.

**Comment by necromanceranne** [comment]
> i hate balance discussions but balance prs are themselves not a threat to maintainability like features and are (usually) short, i dont see why it should be -8.

I think the justification is people might start to put up a lot of balance prs constantly, and the weight of maintainers to have to decided whether or not those balance decisions are right for the game in isolation might make it pretty annoying to have to deal with those prs regularly while pushing an a maintenance focused/code improvement focus.

The price tag is literally because most maintainers do not like balance discussions, which is what every balance pr becomes.

**Comment by nianjiilical** [comment]
it might be worth splitting into major balance and minor balance tags tbh

i spent -8 gbp the other week making a one line change to an antag's min pop requirement and while that makes sense for something like overhauling atmos gases or significantly changing security's guns, it felt a little silly

**Comment by Kylerace** [comment]
Trollface

**Comment by Iamgoofball** [comment]
my gbp you monster

**Comment by LemonInTheDark** [comment]
@tgstation/commit-access 
hi

**Comment by AnCopper** [comment]
Great British Pounds?

**Comment by Fikou** [comment]
-5 seems better for this. it still costs you to create a fix+balance pr, but it wont put you in debt so fast

**Comment by Farquaar** [comment]
> Great British Pounds?

Good boy points
You can exchange them for chicken tendies if you stock up enough of them

**Comment by Kylerace** [comment]
I don't mind if we decide on a different new value, anywhere between -6 and -3 seems goos

**Comment by Kylerace** [comment]
Fucking phone

**Comment by Mothblocks** [comment]
> -5 seems better for this. it still costs you to create a fix+balance pr, but it wont put you in debt so fast

+1

**Comment by Time-Green** [comment]
> +1

@Iamgoofball You heard the man!

**Comment by Iamgoofball** [comment]
@Fikou What's wrong with 1 fix = 1 balance change?

**Comment by ArcaneMusic** [comment]
It's not that balance PRs bad here, goofball. The issue here as I see it is we don't want to spend weeks at a time going back and forth between merged balance PRs when instead we could make balance PRs more balance/tweak heavy to the devleopment cycle.

Maybe 100 credits is too much to charge crew for insulated gloves, so we tweak it down to 25 credits. Then we learn everyone has insulated gloves, so we raise it back up to 75. Back down to 50. 65. 60. Instead, having a high cost on balance PRs encourages contributors to find that middle ground first, and test their code more heavily between testmerges/overarching design.  (replace credits with material cost/power drain/time investment).

I'd be down for 5 GBP.

**Comment by ExcessiveUseOfCobblestone** [comment]
id even be ok with 2 so that way multiple fixes reward you with "bonus" balance credits, if theres an issue where the balance is so drastic that it is basically feature-adding then the system for account for that imo.

the gbp system doesnt save us from not having to talk about balance since there is no cost to PR, only to merge.

**Comment by tralezab** [comment]
-5 is the limits of what i'm ok with

**Comment by ShizCalev** [comment]
-4 / -5 imo

File: .github/gbp.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ reset_label = "GBP: Reset"
 "Accessibility" = 3
 "Administration" = 2
 "Atomic" = 2
-"Balance/Rebalance" = -8
+"Balance/Rebalance" = -3
```
Comment (Mothblocks) [inline .github/gbp.toml:8]:
```suggestion
"Balance/Rebalance" = -5
```

Comment (Kylerace) [reply .github/gbp.toml:8]: after seeing a dumb balance change i now agree with -5

Comment (MrMelbert) [reply .github/gbp.toml:8]: Agree with -5

**Pull request merged**